docs/hopsworksai/aws/cluster_creation.md

```diff
@@ -20,56 +20,86 @@ Select the *Region* in which you want your cluster to run (1), name your cluster
+
+!!! note
+    The overprovisioning will not be taken into account if you set the minimum number of workers to 0 and no resources are used in the cluster. This ensures that the number of nodes can fall to 0 when no resources are used. The overprovisioning will start to take effect as soon as you start using resources.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/create-instance-workers-autoscale.png">
+      <img style="border: 1px solid #000;width:700px;width:506px" src="../../../assets/images/hopsworksai/create-instance-workers-autoscale.png" alt="Create a Hopsworks cluster, autoscale workers configuration">
+    </a>
+    <figcaption>Create a Hopsworks cluster, autoscale workers configuration</figcaption>
+  </figure>
+</p>
+
+### Step 4 select an SSH key
 
 When deploying clusters, Hopsworks.ai installs an ssh key on the cluster's instances so that you can access them if necessary.
 Select the *SSH key* that you want to use to access cluster instances. For more detail on how to add a shh key in AWS refer to [Create an ssh key](getting_started.md#step-3-create-an-ssh-key)
 
 <p align="center">
   <figure>
     <a  href="../../../assets/images/hopsworksai/aws/connect-aws-ssh.png">
-      <img style="border: 1px solid #000" src="../../../assets/images/hopsworksai/aws/connect-aws-ssh.png" alt="Choose SSH key">
+      <img style="border: 1px solid #000;width:700px" src="../../../assets/images/hopsworksai/aws/connect-aws-ssh.png" alt="Choose SSH key">
     </a>
     <figcaption>Choose SSH key</figcaption>
   </figure>
 </p>
 
-### Step 4 select the Instance Profile:
+### Step 5 select the Instance Profile:
 
 To let the cluster instances access the S3 bucket we need to attach an *instance profile* to the virtual machines. In this step, you choose which profile to use. This profile needs to have access right to the *S3 bucket* you selected in [Step 2](#step-2-setting-the-general-information). For more details on how to create the instance profile and give it access to the S3 bucket refer to [Creating an instance profile and giving it access to the bucket](getting_started.md#step-22-creating-an-instance-profile-and-giving-it-access-to-the-bucket)
 
 <p align="center">
   <figure>
     <a  href="../../../assets/images/hopsworksai/aws/connect-aws-profile.png">
-      <img style="border: 1px solid #000" src="../../../assets/images/hopsworksai/aws/connect-aws-profile.png" alt="Choose the instance profile">
+      <img style="border: 1px solid #000;width:700px" src="../../../assets/images/hopsworksai/aws/connect-aws-profile.png" alt="Choose the instance profile">
     </a>
     <figcaption>Choose the instance profile</figcaption>
   </figure>
 </p>
 
-### Step 5 set the backup retention policy:
+### Step 6 set the backup retention policy:
```

Comment:
```suggestion
### Step 6 set the backup retention policy
```

Reply:
In the rest of the docs we have links to `cluster_creation.md#step-5-set-the-backup-retention-policy` It needs to change now to `step-6`